src/components/ModalExport.tsx

```diff
@@ -11,13 +9,21 @@ import { exportContext, getDescendants, getAllChildren, simplifyPath, theme } fr
 import Modal from './Modal'
 import DropDownMenu from './DropDownMenu'
 import LoadingEllipsis from './LoadingEllipsis'
+import ChevronImg from './ChevronImg'
 import { State } from '../util/initialState'
 import { ExportOption } from '../types'
 import useOnClickOutside from 'use-onclickoutside'
 
+interface AdvancedSetting {
+  id: string,
+  onChangeFunc: () => void,
+  defaultChecked: boolean,
+  title: string,
+  description: string,
+}
+
 const exportOptions: ExportOption[] = [
-  { type: 'text/plain', label: 'Plain Text (lossless)', extension: 'txt' },
-  { type: 'text/plain', label: 'Plain Text', extension: 'txt', excludeMeta: true },
```

Comment:
You can now remove `excludeMeta` from the `ExportOption` interface itself since it is no longer being used:

https://github.com/cybersemics/em/blob/84ec54dec3133bfa2e8e2d0827efc4b2ad40ab5f/src/types.ts#L93-L99

Reply:
Done